app/src/main/assets/dark.css

```diff
@@ -233,6 +233,11 @@ blockquote, pre {
   text-justify: none;
 }
 
+#article blockquote p, #article pre p {
+  background: inherit;
+  color: inherit;
```

Comment:
In the first image: Text color with black background is different from the text color with gray background. It's hard to see but I have good eyes :D